### ISSUE TITLE
Show line numbers for code snippets

### ISSUE THREAD

**Issue by jimgymjim** [telegramdesktop/tdesktop#27019]
### Is your feature request related to a problem?

Currently there is no line number in code snippets, adding line number in code snippets would be a great feature.

### Describe the solution you'd like

Adding line numbers in the left side of each line like IDEs on code snippets.

### Describe alternatives you've considered

Nothing.

### Additional context

Link to specific line of code, starting line numbers from custom number which user set also are useful features.

**Comment by Adam-Kay** [comment]
bump. I think this would be useful also

**Comment by georgesglynn** [comment]
Agreed! First thing I noticed when testing code snippets was the lack of line numbers
